### DISCUSSION TITLE
Themeisle Offer Not Working And Support Is not grasping the urgency of matter.

### DISCUSSION THREAD

**Discussion by maggyium** [community/community#130373] [GitHub Education] [answered]
### Hi everyone,

![image](https://github.com/community/community/assets/146563800/5f32be1f-1069-40ff-aacc-76d2fae93d8d)
Github Student Developer Pack has this themeisle offer that hasn't worked since over a month ago when I originally tried contacting their support.
![image](https://github.com/community/community/assets/146563800/f811ce7f-ac97-4810-86d9-7a133d65d897)
It gives this error When trying to claim the offer.
As we aren't yet their customers we aren't allowed to contact their support and github support wont handle tickets regarding these third party offers.
I posted this in their community forum and they said this : 
![image](https://github.com/community/community/assets/146563800/c23c93df-34af-4fbd-a08d-d260b1ada1c1)
This is their last response from May 26 and I haven;t received any updates after it.

**Comment by selul** [comment, marked as answer]
Hi, 

This is Marius from Themeisle. We apologize for the issue you encountered with our GitHub integration. The problem arose due to recent changes in the GitHub Education API.

We reached out to GitHub for clarification and, after a period of communication, we were able to work together to resolve the issue.

A few days ago, the integration was fixed, and it should be working properly. Thank you for your patience.

**Reply by maggyium** [reply]
Hello Maurius ,
Thanks for the update.
As the offer states we get free access to Neve Agency for a year, is it necessary for us to have our credit card on file from the beginning as most students might not have the ability to procure a credit card?

**Reply by selul** [reply]
Hi, yes, for now, it is necessary as the offer is for one year only and is necessary for the renew.
